gulpfile.js

```diff
@@ -263,6 +265,7 @@ function dist() {
         './js/vendor/three.js',
         './js/vendor/three.min.js',
         './js/vendor/webm-writer-0.1.1.js',
+        './js/vendor/lodash.min.js',
```

Comment:
Adding third party libraries directly to the repository is legacy from before we were using npm. New third party dependencies should be added through npm.